README.md

```diff
@@ -227,4 +227,14 @@ beforeEach(
     fixture.detectChanges();
   })
 );
+```
+
+## Using scripts/resources.js file
+
+This file is used to convert the source App Icon and Splash screen image to the target App Icon and Splash screen image size for both IOS and Android mobile's for any responsive and version's. It contains all size target and source images name in it. The command is added in the package.json file in Script tag.
+
+```
```

Comment:
Something odd has happened here, which is causing an empty code block to render below this text. See the screenshot below. Can you fix this?

![image](https://user-images.githubusercontent.com/7593323/150455515-ad0cbbf3-7872-4039-83b4-ff11d3bcc175.png)

Reply:
Ok, I will check and fix the issue